src/lib/App.js

```diff
@@ -75,9 +75,5 @@ export default class App extends Component {
 }
 
 App.propTypes = {
-  config: PropTypes.object,
-};
-
-App.defaultProps = {
-  config: {},
+  config: PropTypes.object.isRequired,
```

Comment:
As I understand this, your Application will not start if you don't provide a config.
IMHO it should be required if we have any required config settings for the App to work, and these props should be explicitly specified in expected PropTypes. If not then it be just empty `{}` if the used doesn't provide one.